File: Chmiel_zadania na lekcji/warunek/warunek.cpp
/*
 * warunek.cpp
 * 
 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	int a, b, c;
    a = b = c = 0;
    
    cout << "Podaj pierwszą liczbę "; 
    cin >> a;
    cout << "Podaj drugą liczbę "; 
    cin >> b;
    cout << "Podaj trzecią liczbę ";
    cin >> c;
    
    if (a>b and a>c) cout << "Największa jest" << a << endl;
    if (b>a and b>c) cout << "Największa jest" << b << endl;
    else cout << "Największa jest" << c << endl; 
        
    return 0;
}

File: Chmiel_zadania na lekcji/warunek/warunek.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  warunek.py
#  


def max(a, b, c):
    
    maks = a
    
    if b > maks:
        maks = b
        
    if c > maks:
        maks = c
    
    return  maks 
    
def main(args):
    
    a = int(input("Podaj pierwszą liczbę: " ))
    b = int(input("Podaj drugą liczbę: " ))
    c = int(input("Podaj trzecią liczbę: " ))
    maks = najwieksza(a, b, c)
    
    print("Najwiekszą liczbą jest ", maks)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
